polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1184,7 +1184,6 @@ impl parachains_slashing::Config for Runtime {
 		ReportLongevity,
 	>;
 	type WeightInfo = parachains_slashing::TestWeightInfo;
-	type BenchmarkingConfig = parachains_slashing::BenchConfig<200>;
```

Comment:
The overall observation is that we had to add temporarily a lot of bounds to pallets just for benchmarks, but as staking has now proper bounds for all its storage, they are not needed ✅